src/library/searchqueryparser.h

```diff
@@ -11,14 +11,19 @@
 
 class SearchQueryParser {
   public:
-    SearchQueryParser(QSqlDatabase& database);
+    SearchQueryParser(TrackCollection* pTrackCollection);
+    SearchQueryParser(QSqlDatabase& database)
```

Comment:
Maybe the test for searchqueryparser should take a TrackCollection instead of just a database connection.